src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```

Comment:
```suggestion
        private final Random random = new Random();
```
and add it on the next line after class

Reply:
Modifier 'private' not allowed here:7